src/main/java/uk/gov/hmcts/reform/hmc/model/RequestDetails.java

```diff
@@ -4,13 +4,12 @@
 import lombok.NoArgsConstructor;
 import uk.gov.hmcts.reform.hmc.exceptions.ValidationError;
 
-import java.time.LocalDateTime;
 import javax.validation.constraints.NotNull;
 
 @Data
 @NoArgsConstructor
 public class RequestDetails {
 
-    @NotNull(message = ValidationError.REQUEST_TIMESTAMP_NULL_EMPTY)
-    private LocalDateTime requestTimeStamp;
+    @NotNull(message = ValidationError.VERSION_NUMBER_NULL_EMPTY)
+    private Integer versionNumber;
```

Comment:
There was previously an `UpdateRequestDetails` class which was used only for the PUT and extended this. Deleted that one under this PR as `RequestDetails` is now only needed for the PUT and only needs `versionNumber`.